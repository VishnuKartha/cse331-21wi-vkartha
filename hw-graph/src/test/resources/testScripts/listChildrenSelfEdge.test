# listing the children of the parent node in a graph where there is a self edge

# Create a Graph
CreateGraph graph1

# Add a nodes
AddNode graph1 n1

# adds a self edge
AddEdge graph1 n1 n1 e1

# List the children
ListChildren graph1 n1
