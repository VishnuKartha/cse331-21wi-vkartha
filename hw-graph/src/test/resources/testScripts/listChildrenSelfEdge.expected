# listing the children of the parent node in a graph where there is a self edge

# Create a Graph
created graph graph1

# Add a nodes
added node n1 to graph1

# adds a self edge
added edge e1 from n1 to n1 in graph1

# List the children
the children of n1 in graph1 are: n1(e1)
